test/integration/cloudresource_e2e_test.go

```diff
@@ -27,9 +27,11 @@ import (
 	"github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 
 	"antrea.io/nephe/apis/crd/v1alpha1"
+	cpaUtils "antrea.io/nephe/pkg/cloud-provider/utils"
```

Comment:
nit: no camelcase in imports.. use cpautils

Reply:
Done